docs/cloud/satellites/best-practices.md

```diff
@@ -0,0 +1,90 @@
+# Satellites Best Practices
+
+Earthly Satellites are remote runner instances provided by the Earthly team, offering the ability to perform builds in the cloud while retaining cache between runs. This best practices guide aims to help users optimize their workflow and leverage the full potential of Earthly Satellites.
+
+## Size Satellites for Builds
+
+Properly sizing Earthly Satellites is crucial to achieving optimal performance and resource utilization. Consider the following when determining which Satellite size is appropriate for your use case.
```

Comment:
"crucial to achieving optimal performance" is very wordy, can we condense this sentence?

Reply:
maybe even replace this entirely with:

```suggestion
Consider the following when picking a satellite size:
```